root/edit/details/AddStandaloneRecording.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import ExpandedArtistCredit from '../../components/ExpandedArtistCredit';
+import formatTrackLength from
+  '../../static/scripts/common/utility/formatTrackLength';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type AddStandaloneRecordingProps = {
+  +display_data: {
+    +artist_credit: ArtistCreditT,
+    +comment: string | null,
+    +length: number | null,
+    +name: string,
+    +recording: RecordingT,
+    +video: boolean,
+  },
+};
+
+const AddStandaloneRecording =
+  ({edit}: {edit: AddStandaloneRecordingProps}) => {
+    const display = edit.display_data;
```

Comment:
I'd suggest something like
```suggestion
const AddStandaloneRecording = (
  {edit}: {+edit: AddStandaloneRecordingProps}
) => {
  const display = edit.display_data;
```
because that way you only have to indent 2 spaces instead of 4